unitsnet_py/abstract_unit.py

```
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from functools import total_ordering
+
+
+@total_ordering
+class AbstractMeasure:
+    _value: float
+
+    def __str__(self):
+        return self.to_string()
+
+    def __add__(self, other: AbstractMeasure):
+        if not isinstance(other, type(self)):
+            return NotImplemented
```

It should not be a `raise TypeError("unsupported operand....` ?

---

This is what happens when we use ``return NotImplemented``: the comparison is delegated to ``other`` object.
If the ``other`` object cannot perform the comparison as well, then ``TypeError`` is raised.
This is common in Python to implement comparisons this way.

References:
- https://stackoverflow.com/questions/1062096/python-notimplemented-constant
- https://docs.python.org/3.6/library/constants.html#NotImplemented